index.html

```diff
@@ -416,15 +423,21 @@ <h4>Audio Description Examples</h4>
         <h4>Dubbing Examples</h4>
         <p>From the basic structure of <a href="#example-1"></a>, a transcription of the audio programme produces an original language dubbing <a>script</a>,
           which can look as follows. No specific style or layout is defined, and here the focus is on the transcription of the dialogs.
-          Characters are identified within the <code>&lt;metadata&gt;</code>.</p>
+          Characters are identified within the <code>&lt;metadata&gt;</code>.
+          Note that the language and the <a>text language source</a> are defined using
+          <code>xml:lang</code> and <a><code>daptm:langSrc</code></a> respectively,
+          which have the same value
+          because the transcript is not translated.</p>
         <pre class="example"
           data-include="examples/intro-original-language.xml"
           data-include-format="text">
         </pre>
 
         <p>After translating the text, the document is modified. It includes translation text, and
-        in this case the original text is preserved. The main document language is changed to indicate
-        that the focus is on the translated language:</p>
+        in this case the original text is preserved. The main document's <a>default language</a> is changed to indicate
+        that the focus is on the translated language.
+        The combination of the <code>xml:lang</code> and <a><code>daptm:langSrc</code></a> attributes are used
+        to mark the text as being original or translated:</p>
```

Comment:
I note that Example 9 and 10 are not using inheritance of xml:lang nor daptm:langSrc. This is fine. I just wonder if we should say, the xml:lang and daptm:langSrc have been duplicated here for readability but could be omitted in some places following the inheritance model.

Reply:
Done in a1c80d4